dist/src/main/config/broker.yaml.template

```diff
@@ -201,6 +201,100 @@
       # This setting can also be overridden using the environment variable ZEEBE_BROKER_THREADS_IOTHREADCOUNT
       # ioThreadCount: 2
 
+    # backpressure:
```

Comment:
This whole block (unfortunately) needs to be copied into:

`brokerstandalone.yaml.template` Same thing, but the template for a standalone broker. And I guess we want to enable backpressure there, too.